src/components/venuePortal.js

```diff
@@ -3,37 +3,24 @@ import SignUp from './venueSignup'
 import SignIn from './venueSignin'
 import VenueProfile from './venueProfile'
 
-
 export default class VenuePortal extends Component {
   constructor() {
     super();
     this.state = {
-      signup: false,
-      signin: false,
-      loggedIn: false,
-      venue: null
+      process: false
       }
       this.handler = this.handler.bind(this);
     }
-
-    _onSignupClick(){
-      this.setState({
-        signin: false,
-        signup: true
-      })
-    }
-
-    _onSignInClick(){
+   
+    _changeProcess(process){
       this.setState({
-        signup: false,
-        signin: true
+        process: process,
       })
     }
 
     _onLogoutClick() {
       this.setState({
-        signup: false,
-        signin: false,
+        process: false,
```

Comment:
Might just be a stylistic thing but I find it much harder to read the code when all of the state's for a component aren't declared in the constructor - not sure of the conventions on this one